sessionctx/variable/sysvar.go

```diff
@@ -76,6 +76,11 @@ var defaultSysVars = []*SysVar{
 	/* TiDB specific variables */
 	{Scope: ScopeNone, Name: TiDBEnableEnhancedSecurity, Value: Off, Type: TypeBool},
 	{Scope: ScopeNone, Name: TiDBAllowFunctionForExpressionIndex, ReadOnly: true, Value: collectAllowFuncName4ExpressionIndex()},
+	{Scope: ScopeNone, Name: TiDBStmtSummaryEnablePersistent, Value: Off, Type: TypeBool, ReadOnly: true},
+	{Scope: ScopeNone, Name: TiDBStmtSummaryFilename, Value: DefTiDBStmtSummaryFilename, Type: TypeStr, ReadOnly: true},
+	{Scope: ScopeNone, Name: TiDBStmtSummaryFileMaxDays, Value: strconv.Itoa(DefTiDBStmtSummaryFileMaxDays), Type: TypeInt, ReadOnly: true},
+	{Scope: ScopeNone, Name: TiDBStmtSummaryFileMaxSize, Value: strconv.Itoa(DefTiDBStmtSummaryFileMaxSize), Type: TypeInt, ReadOnly: true},
+	{Scope: ScopeNone, Name: TiDBStmtSummaryFileMaxBackups, Value: strconv.Itoa(DefTiDBStmtSummaryFileMaxBackups), Type: TypeInt, ReadOnly: true},
```

Comment:
If indeed we need to provide variables, it seems that get functions haven't been implemented.

Reply:
There is a unit test to enforce that the variables and configs match, so we have to add these variables. But they are not runtime configurable, so I guess just setting `Value` is enough. I tested that it can be read:
```
mysql> select @@tidb_stmt_summary_filename;
+------------------------------+
| @@tidb_stmt_summary_filename |
+------------------------------+
| tidb-statements.log          |
+------------------------------+
```

Reply:
What happens if setting another filename?

Reply:
```
mysql> set global tidb_stmt_summary_filename = 'a.log';
ERROR 1238 (HY000): Variable 'tidb_stmt_summary_filename' is a read only variable
```

Reply:
Oh! I should use the value from `config` package instead of `DefTiDBStmtSummaryXxx`.